README.md

```diff
@@ -1,29 +1,32 @@
 NServiceBus.RavenDB
 ======================
 
-Persistence support for NServiceBus RavenDB
+Persistence support for NServiceBus.RavenDB
 
 ## How to use this library
 
-After adding a reference to it from your project, simply specify RavenDB to be used for persistence, and set a RavenDB connection string in your `app.config` as `NServiceBus/Persistence/RavenDB` or pass the conection parameters like this:
+Add the [NuGet](https://www.nuget.org/packages/NServiceBus.RavenDB/) package to your project.
+Specify `RavenDB` to be used for persistence, and set a RavenDB connection string in your `app.config` as `NServiceBus/Persistence/RavenDB` or pass the connection parameters like this:
 
```

Comment:
Suggestion: Specify `RavenDB` as the persistence mechanism, and set a ...

(In general, I try to go for an active voice in how-to docs.)